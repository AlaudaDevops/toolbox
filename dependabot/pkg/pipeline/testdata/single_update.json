{
    "projectPath": "/test/path",
    "totalPackages": 1,
    "successfulUpdates": [
        {
            "package_name": "golang.org/x/crypto",
            "current_version": "v0.0.1",
            "fixed_version": "v0.1.0",
            "vulnerability_ids": [
                "CVE-2023-1234"
            ],
            "language": "go",
            "success": true,
            "error": ""
        }
    ],
    "failedUpdates": [],
    "summary": "Updated 1 package successfully"
}