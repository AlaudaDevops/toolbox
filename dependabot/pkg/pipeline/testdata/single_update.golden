fix: update golang.org/x/crypto from v0.0.1 to v0.1.0

Fixes security vulnerabilities: [CVE-2023-1234]
